tar/tarsnap.1-man.in

```diff
@@ -752,8 +739,14 @@ option to
 \fBfind\fP(1).
 .TP
 \fB\--null-output\fP
-(list-archives mode only)
-Archive names in output are separated by null characters, not by newlines.
+(x, t, and list-archives modes only)
+Archive names and filenames in output are separated by null characters, not by
+newlines.
+.PP
+If there are multiple fields on a line, they will be separated by two null
+characters.
+If a command line is printed within one of those fields, each argument will be
+separated by three null characters.
```

Comment:
re: "3 NULs"... this is part of the first commit, "generalize explanation of `--null-output`", which applies to the already-implemented `--list-archives`.

I considered making a PR just for that 1 commit (since it's "cleaning up an existing feature" rather than "adding a new feature"), but I figured that you'd rather not bother having a separate PR for that.

Reply:
Ah right, that bit landed in 2023.